Cargo.toml

```diff
@@ -41,28 +41,27 @@ harness = false
 # match exactly, since the build.rs uses the crate itself as a library.
 
 [dependencies]
-byteorder = {version = "1", default-features = false }
+rand = { version = "0.5.0-pre", default-features = false, git = "https://github.com/rust-lang-nursery/rand" }
```

Comment:
So tor is currently pinned to rand-0.5.0-pre.1 because I needed to implement an RNG using rand_core, which then required using 0.5.0… so this is still not going to resolve to the same trait, making my current task (https://bugs.torproject.org/23886) impossible. Probably I should go ask nicely for another pre-release and we should pin to that before releasing curve25519-dalek-0.17?

Reply:
Okay, it's done already in https://github.com/rust-lang-nursery/rand/issues/447! I'll fix up your branch to use `rand-0.5.0-pre.1`.

Reply:
I think it should be `0.5.0-pre.2`.